examples/kitchen-sink.yaml

```diff
@@ -379,3 +379,15 @@ resource:
     service.name: !!str "unknown_service"
   # Configure the resource schema URL.
   schema_url: https://opentelemetry.io/schemas/1.16.0
+  # Configure resource detectors.
+  detectors:
+    # Configure disabled resource attribute keys provided by resource detectors.
+    #
+    # Attribute keys are evaluated to match disabled_attribute_keys in the following manner:
+    #   * If the value of the attribute key exactly matches.
+    #   * If the value of the attribute key matches the wildcard pattern, where '?' matches any single character and '*' matches any number of characters including none.
+    disabled_attribute_keys:
+      # Configure exact match disabled attribute key.
+      - process.command_args
+      # Configure wildcard match disabled attribute key(s).
+      - process.*
```

Comment:
Discussed in the 1/8/24 configuration SIG:

- Decided to move this property up to the top level instead of nesting under detectors. This solves any ambiguity issues about whether attributes like `telemetry.sdk.*` are included in the set, since those may be part of the default resource or provided by resource detectors. 
- Discussed whether wildcard syntax is sufficient. Opened #55 to discuss. I propose we stick with wildcard syntax for now and use user feedback to determine if full regex support is needed before a 1.x release.
- Discussed whether it would also be useful to match on resource values. Answer is probably? Could add `disabled_attribute_values` in a separate PR.